tests/docker/php/cli/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM php:8.1-cli
+
+MAINTAINER  A. Egorov <anat-gor@ya.ru>
+
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+RUN chmod +x /usr/local/bin/install-php-extensions
+
+
+RUN install-php-extensions redis intl pcntl xdebug @composer
+RUN rm  -rf /tmp/*
+
+ENV XDEBUG_MODE=coverage
```

Comment:
Could you switch to [codecov.io](https://docs.codecov.com/docs/testing-with-docker#codecov-outside-docker), that's used in the rest yii3 repos?